docs/docusaurus/docs/core/connect_to_data/sql_data/_create_a_data_source/_datasource_method_reference_table.md

```diff
@@ -2,6 +2,6 @@
 |------------------|------------------------------------------------------------------------------|
 | PostgreSQL       | `context.data_sources.add_postgres(name: str, connection_string: str)`       |
 | SQLite | `context.data_sources.add_sqlite(name: str, connection_string: str)`         |
-| DataBricks SQL   | `context.data_sources.add_databricks_sql(name: str, connection_string: str)` |
 | Snowflake        | `context.data_sources.add_snowflake(name:str, connection_string: str)`       |
+| DataBricks SQL   | `context.data_sources.add_databricks_sql(name: str, connection_string: str)` |
```

Comment:
note for reviewer: I moved this so the order of databases is consistent across both tables on this page